README.md

```diff
@@ -1 +1,33 @@
-# curated-authors-transformer
\ No newline at end of file
+# curated-authors-transformer
+
+[![CircleCI](https://circleci.com/gh/Financial-Times/curated-authors-transformer.svg?style=svg)](https://circleci.com/gh/Financial-Times/curated-authors-transformer)
+
+Retrieves authors data curated by editorial people and transforms it to People according to UP json model.
+The authors data is specified by a Google spreadsheet which is accessible through Bertha APIs ()
+The service exposes endpoints for getting all the cureted authors UUIDs and for getting authors by uuid.
+
```

Comment:
Fixed it!
I put the brackets because my intension was to put the link to bertha docs